server/src/controllers/beneficiary.controllers.ts

```diff
@@ -0,0 +1,45 @@
+import Beneficiary from "../models/BeneficiaryModel";
+import { Request, Response } from 'express';
+
+const editBeneficiary = async (req: Request, res: Response): Promise<Response> => {
+    try {
+        const beneficiaryID = req.params.id; // Use params instead of body for ID
+        if (beneficiaryID) {
+            const result = await Beneficiary.findByIdAndUpdate(
+                beneficiaryID,
+                req.body,
```

Comment:
This isn't type safe. You should import the interface defined in `BeneficiaryModel.ts` and use `req.body as IBeneficiary`. See example here (https://github.com/ChangePlusPlusVandy/vac-net/blob/a9c4e9bab9f8ff75490e1b4e561bcb6290a39fbc/server/src/controllers/session.controllors.ts#L7)